charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions
+
+Data contribution: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data contribution become publicly available
+Data distribution: the packaging of metadata and data from multiple data contributions in meaningful ways and any announcements or documentation supporting it
+
+## Objectives
+
+The objective of the Data Operations is to provide the scientific community with effective tools and resources from the DCP which maximize the value and use of HCA data
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
+  - Coordinate DCP-wide end-to-end testing pre-release and continued QC checks post-release to ensure data and data resource integrity and useability through component updates
+  - Identify data release- and data distribution-associated engineering needs of all DCP components and participate in crafting technical specifications
+  - Establish and integrate into the DCP roadmap a reproducible plan that includes the specifications for and the timing of iterative data distribution to the community
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
+  - Engage with the [Compliance Working Group](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md) to ensure that the data release and data distribution processes are facilitating appropriate data access to all of the types of consumers
```

Comment:
We tend to use the term responsible in this context so perhaps something like

Engage with the compliance working group to ensure the our data release and distribution processes ensure responsible HCA data sharing to all types of consumers

Reply:
Incorporated in 38f0488